clab/config.go

```diff
@@ -321,24 +335,24 @@ func (c *CLab) verifyLinks() error {
 }
 
 // VerifyImages will check if image referred in the node config
-// either pullable or present or is available in the local registry
-// if it is not available it will emit an error
+// either pullable or is available in the local image store
 func (c *CLab) VerifyImages(ctx context.Context) error {
-	images := map[string]struct{}{}
+
+	images := make(map[string]string)
+
 	for _, node := range c.Nodes {
-		// skip image verification for bridge kinds
-		if node.Config().Kind == "bridge" || node.Config().Kind == "ovs-bridge" {
-			continue
-		}
-		if node.Config().Image == "" {
-			return fmt.Errorf("missing required image for node %s", node.Config().ShortName)
+
+		for _, imageName := range node.GetImages() {
+			if imageName == "" {
+				log.Errorf("missing required image for node %q", node.Config().ShortName)
```

Comment:
shouldn't we return the error here instead of logging it?
if one of the required images is missing, the node has no chances to start, so we should stop here?

Reply:
For some reason, I thought that there could be nodes with no images. Like bridge?

Reply:
in that case the nil will be the vaule of that slice, and the range won't engage?

Reply:
I've re-worked this to return error instead. Inside `GetImages` I make sure I always return empty map for nodes that have no images.